src/main/resources/hackathon-data/municipalities.csv

```diff
@@ -0,0 +1,85 @@
+id,municipality,court_id

```

Comment:
All the CSV files should have been removed in favor of just SQL Insert Statements.....